nfadeeva/assignment5/find_alphabet.py

```diff
@@ -0,0 +1,83 @@
+"""
+Find the alphabet on the given dictionary + identify inconsistent dictionaries
+"""
+
+from typing import List
+from collections import defaultdict
+from collections import deque
+
+
+class Graph:
+    def __init__(self):
+        self.adj = defaultdict(list)
```

Comment:
avoid using abbreviations: https://github.com/google/styleguide/blob/gh-pages/pyguide.md#s3.16-naming

Reply:
@aliskin thank you for your comments, fixed all places, check it)